.drone.yml

```diff
@@ -797,6 +797,12 @@ services:
   volumes:
   - name: mysql
     path: /var/lib/mysql
+- environment: {}
+  image: redis:6.2.1-alpine
```

Comment:
this is a 2 year old version, should we be using 6.2.11 (latest 6.2.x) or 7.0.10 (current stable) instead?

Reply:
it seems that in cloud we use [another image](https://github.com/grafana/deployment_tools/blob/28f2a0d86338e8150a57814fdfa0e1c53461db8a/ksonnet/lib/hosted-grafana/images.libsonnet#L21)
it would make sense to use the same, @grafana/grafana-delivery WDYT?

Reply:
i think it makes sense to use the same.

Reply:
I have ended up using the `redis:6.2.11-alpine` for now because the image used by cloud [requires extra setup for setting a password](https://drone.grafana.net/grafana/grafana/109102/8/4) so it requires extra work for modifying the `REDIS_URL` for test to include it